### ISSUE TITLE
KLpq normalization

### ISSUE THREAD

**Issue by tomisilander** [blei-lab/edward#752]
In tutorial http://edwardlib.org/tutorials/klpq and at http://edwardlib.org/api/ed/KLpq (build_loss_and_gradients) it looks like there is a superfluous "normalisation" (division by S) that is not needed (i.e. wrong) since the importance weights have already been normalized.
 
The code https://github.com/blei-lab/edward/blob/master/edward/inferences/klpq.py
may have the same problem since it is using tf.reduce_mean (instead of just sum).

**Comment by dustinvtran** [comment]
I agree and also think it's a change from reduce_mean to reduce_sum. 

Computationally because we always use gradient descent, the additional division can be thought of as scaling down the learning rate. So any current experiments with `ed.KLpq` are still valid.

**Comment by dustinvtran** [comment]
Fixed in #765.
